gnosis/safe/api/base_api.py

```diff
@@ -24,7 +25,7 @@ def __init__(
         network: EthereumNetwork,
         ethereum_client: Optional[EthereumClient] = None,
         base_url: Optional[str] = None,
-        request_timeout: int = 10,
+        request_timeout: int = int(os.environ.get("SAFE_BASE_API_REQUEST_TIMEOUT", 10)),
```

Comment:
I think that would be better define this timeout in the childs, intead here. 
I mean something like `SAFE_TRANSACTION_SERVICE_REQUEST_TIMEOUT`, and the same for the relay, but not sure if will be continued.

Reply:
Relay is discontinued, we should remove it. But I agree on defining this variable for every child

Reply:
Added in TransactionServiceApi. RelayServiceApi will be removed in this [PR](https://github.com/safe-global/safe-eth-py/pull/949).